### PR TITLE
Add Python 3 to build files.

### PR DESCRIPTION
**Pull request by fruffy**
This pull request adds Python 3 to the third party dockerfile. This change should no effect of any downstream projects since the original Python 2 dependencies are not touched.

### CODE REVIEW

**Comment by fruffy** [comment]
Once Python3 is supported by all downstream dependencies we can also remove Python 2 here?

File: Dockerfile [hunk 1]
```diff
@@ -50,30 +49,36 @@ RUN make DESTDIR=/output install
 FROM ubuntu:16.04 as scapy-vxlan
 ARG DEBIAN_FRONTEND=noninteractive
 ARG MAKEFLAGS=-j2
-ENV SCAPY_VXLAN_DEPS python python-pip python-setuptools
+ENV SCAPY_VXLAN_DEPS python python3 python-dev python3-dev \
```
Comment (antoninbas) [inline Dockerfile:52]: why add `python-dev` / `pyhton3-dev` here?

**Review by antoninbas — COMMENTED** [review]
This looks good to me. Since we are not removing any Pyhton2 software, I am hoping this doesn't break any downstream build. We can make adjustments after merging this as we add Python3 support to other projects.

Adding @pkotikal for review.

**Review by pkotikal — APPROVED** [review]

**Review by antoninbas — APPROVED** [review]
LGTM - I will merge it and update the stable branch as well, which is the one consumed by other p4lang projects IIRC

**Pull request merged**